app/models.py

```diff
@@ -437,6 +473,21 @@ class UserEvent(Event):
     def from_user(cls, user, **kwargs):
         return cls(user_id=user.id, deployment=user.deployment, **kwargs)
 
+class SimpleUserEvent(UserEvent):
+    '''
+    Any kind of user event that doesn't have any extra data associated
+    with it.
+    '''
+
```

Comment:
Er, note that because we're not specifying a `__tablename__` here, SQLAlchemy will use [single table inheritance](http://docs.sqlalchemy.org/en/rel_0_8/orm/inheritance.html#single-table-inheritance), i.e. the migration will add a column to the table of the superclass (`user_events`) rather than adding a whole new table. Since the only field, `subtype`, is just an integer, this doesn't *seem* like a terrible waste of space, and it also means one less join, but we can change it to use a separate table instead if you want.